exercises/00-init/src/modules/users/users-selectors.js

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from 'reselect'
+
+const getUsers = (state) => state.users
+
+const getTitle = createSelector(
+    getUsers,
+    (state) => state.title,
+)
+
+const getUsersList = createSelector(
+    getUsers,
+    state => state.userList
+)
+
+
+const getUsersUpperCase = createSelector(
+    getUsersList,
+    userList => userList.map( user => (
+        {
+            ...user, 
+            lastName: user.lastName.toUpperCase() 
+        } 
+    ))
+)
+
+export { getTitle, getUsers, getUsersUpperCase} 
```

Comment:
More common way is to export the functions directly, e.g. on line 16 you would have 
```
export const getUsersUpperCase = createSelector(
...
```
Note that it can be only done in browser though, not in node.js, for the same reason why you must use `require` instead of `import .. from` - they use a different module system (if you want to learn more, google "ES6 modules vs CommonJS modules")

Reply:
It feels like the language designers hate having more lines of code than strictly necessary :D. Why declare a function and export it in a different line when you can do it in one? :)

Reply:
That's the spirit! :D